lib/runtime/src/config.rs

```diff
@@ -107,10 +129,20 @@ impl RuntimeConfig {
         Ok(config)
     }
 
+    /// Check if HTTP server should be enabled
+    /// HTTP server is enabled by default, but can be disabled by setting DYN_RUNTIME_HTTP_ENABLED to false
+    /// If a port is explicitly provided, HTTP server will be enabled regardless
+    pub fn http_server_enabled(&self) -> bool {
+        self.http_enabled
+    }
+
     pub fn single_threaded() -> Self {
         RuntimeConfig {
             num_worker_threads: 1,
             max_blocking_threads: 1,
+            http_server_host: DEFAULT_HTTP_SERVER_HOST.to_string(),
```

Comment:
get default builder from figment which parses env, then override the num worker and max blocking threads